### DISCUSSION TITLE
How to set the description of a keymap of which key

### DISCUSSION THREAD

**Discussion by OriLiMu** [LazyVim/LazyVim#1988] [Q&A] [answered]
Now I add leader key like this which cant define the description. I search in the doc not find related , How to fix it?
`keymap("n", "<leader>rr", "<cmd>edit $MYVIMRC<CR>", { silent = true })`

**Comment by dpetka2001** [comment, marked as answer]
Assuming your `keymap` is an alias for `vim.keymap.set` you can set a description for the key with defining `desc = "My keymap"`. See `:h vim.keymap.set()` for more info.
